content/topics/coding-horrors/_index.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Coding Horrors
```

Comment:
It seems to me that this is all TDD topic material. Why not just put it in the TDD topic?